contracts/token/ERC20/extensions/ERC20Wrapper.sol

```diff
@@ -79,4 +85,15 @@ abstract contract ERC20Wrapper is ERC20 {
         _mint(account, value);
         return value;
     }
+
+    /**
+     * @dev Determines whether an account can receive either wrapped or underlying tokens by operating
+     * in this contract.
+     *
+     * By default, it disallows this contract to receive tokens so they don't get stuck, however,
+     * another account can still send tokens to it.
+     */
+    function _canReceiveToken(address account) internal virtual returns (bool) {
```

Comment:
To we have any plan for people to override this function ? IMO it would be simpler/cleaner to do
```solidity
if (account == address(this)) {
  revert ERC20InvalidReceiver(account);
}
```
in both functions and not add this extra piece of code.

Reply:
The initial suggestion from Fran was to use a more general `_canReceiveToken` for every ERC20 but we haven't yet reached consensus with that so I'm implementing this suggestion instead.

Solving